src/ResourceManagement/CosmosDB/Domain/ICosmosDBAccount.cs

```diff
@@ -105,6 +110,22 @@ public interface ICosmosDBAccount :
         /// </summary>
         System.Collections.Generic.IReadOnlyList<Models.VirtualNetworkRule> VirtualNetworkRules { get; }
 
+        /// <param name="name">Name of private endpoint connection.</param>
+        /// <return>The specific private endpoint connection.</return>
+        Microsoft.Azure.Management.CosmosDB.Fluent.IPrivateEndpointConnection GetPrivateEndpointConnection(string name);
+
+        /// <param name="name">Name of private endpoint connection.</param>
+        /// <return>The specific private endpoint connection.</return>
+        Task<Microsoft.Azure.Management.CosmosDB.Fluent.IPrivateEndpointConnection> GetPrivateEndpointConnectionAsync(string name, CancellationToken cancellationToken = default(CancellationToken));
+
+        /// <param name="groupName">Group name of private link resource.</param>
+        /// <return>The specific private link resource group.</return>
+        Microsoft.Azure.Management.CosmosDB.Fluent.IPrivateLinkResource GetPrivateLinkResource(string groupName);
+
+        /// <param name="groupName">Group name of private link resource.</param>
+        /// <return>The specific private link resource group.</return>
+        Task<Microsoft.Azure.Management.CosmosDB.Fluent.IPrivateLinkResource> GetPrivateLinkResourceAsync(string groupName, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Do we have `<summary>` for these methods?

Same for below.

Reply:
these get and list function is summarized in `<return>`, thus no additional `<summary>`